src/components/ui/AntDesign.jsx

```diff
@@ -0,0 +1,267 @@
+                                        avatar={<InfoCircleFilled/>}
+                                        title={props.data.action.location.title}
+                                />
+                            </List.Item>
+                            <List.Item>
+                                <List.Item.Meta 
+                                        avatar={<InfoCircleFilled/>}
+                                        title={props.data.action.activity.title}
+                                />
+                            </List.Item>
+                        </List>
+                        <Link>Edit section</Link>
+                    </Col>
+                </Row>
+            <Row gutter={[0, 40]}>
+                <Col span={12}>
+                    <Space align="center">
+                        <InfoCircleFilled/>
+                        <Title level={3}>Sign-ups</Title>
+                    </Space>
+                </Col>
+                <Col span={12}>
+                    <QuestionCircleFilled style={{color: "grey"}} />
+                    <Text type="secondary">Click in list to book</Text>
+                </Col>
+            </Row>
+            <Row gutter={[0, 40]}>
+                <Col span={24}>
+                    <InfoCircleFilled style={{color: "grey"}} />
+                    <Text type="secondary">No Pending sign-ups to be booked</Text>
+                </Col>
+            </Row>
+            <Row gutter={[0, 40]}>
+                <Col span={24}>
+                    <Space direction="vertical">
+                        <Space align="center">
+                            <InfoCircleFilled/>
+                            <Title level={3}>Contact person</Title>
+                        </Space>
+                        <Space>
+                            <Avatar shape="square" size={50}>
+                                {props.data.person.first_name.charAt(0)}
+                                {props.data.person.last_name.charAt(0)}
+                            </Avatar>
+                            <Space direction="vertical">
+                                <Title level={4}>
+                                    {props.data.person.first_name} {props.data.person.last_name}
+                                </Title>
+                                <Text type="secondary">Drag another person here or select one to change</Text>
+                            </Space>
+                        </Space>
+                    </Space>
+                </Col>
+            </Row>
+            <Row gutter={[0, 40]}>
+                <Col span={24}>
+                    <Space direction="vertical">
+                        <Space align="center">
+                            <InfoCircleFilled/>
+                            <Title level={3}>Booked participants</Title>
+                        </Space>
+                        <Cascader
+                            action={props.data.action}
```

Comment:
What is this?

Reply:
This is a selection element. I think I tried to add the available action here without luck. Can try fix this with the other changes.

Reply:
Looking at the [documentation for Cascader](https://ant.design/components/cascader/), there is no property called `action`. I believe the property you may have been looking for is `value` (`value={props.data.action}`). But even that doesn't look right, because the cascader should be a list of people, and the `value` should be the selected person, not the action.